pkg/executables/govc.go

```diff
@@ -86,11 +86,7 @@ func (g *Govc) Close(ctx context.Context) error {
 		return nil
 	}
 
-	if err := g.Logout(ctx); err != nil {
-		return err
-	}
-
-	return g.Executable.Close(ctx)
+	return g.Logout(ctx)
```

Comment:
It doesn't, I left it public to be able to test it from `executables_test`. And it didn't seem too bad since it's part of the `govc` interface
The other alternative is to create a new tests file on package `executables`